create-hpc-vmss-linux/azuredeploy.json

```diff
@@ -0,0 +1,358 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"VM SKU Type": {
+			"defaultValue": "Standard_HC44rs",
+			"allowedValues": [
+				"Standard_HC44rs",
+				"Standard_HB60rs"
+			],
+			"type": "String",
+			"metadata": {
+				"description": "Which instance would you like to deploy to? Not all VMs are available in all regions"
+			}
+		},
+		"VMSS Name": {
+			"maxLength": 10,
+			"type": "String",
+			"metadata": {
+				"description": "String used as a base for naming resources. Must be 3-10 characters in length and globally unique across Azure. A hash is prepended to this string for some resources, and resource-specific information is appended."
+			}
+		},
+		"Compute Node Image": {
+			"type": "string",
+			"defaultValue": "CentOS-7.6 HPC",
+			"allowedValues": [
+				"CentOS-7.6",
+				"CentOS-7.6 HPC"
+			],
+			"metadata": {
+				"description": "The VM image of the compute nodes"
+			}
+		},
+		"Instance Count": {
+			"maxValue": 300,
+			"type": "Int",
+			"metadata": {
+				"description": "Number of VM instances (300 or fewer)."
+			}
+		},
+		"Username": {
+			"maxLength": 10,
+			"type": "String",
+			"metadata": {
+				"description": "User name."
+			}
+		},
+		"Password": {
+			"maxLength": 32,
+			"type": "securestring",
+			"metadata": {
+				"description": "User password for compute nodes."
+			}
+		},
+		"RSA Public Key": {
+			"type": "String",
+			"metadata": {
+				"description": "The RSA public key to access the headnode."
+			}
+		}
+	},
+	"variables": {
+		"resourceLocation": "[resourceGroup().location]",
```

Comment:
Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

When that's done, set a value for the parameter in the parameters file to westus2 (that's where the CI subscription has quota).

When CI passes we'll do a full review